### DISCUSSION TITLE
Connection of local ODF maxima to Miller crystal planes

### DISCUSSION THREAD

**Discussion by cbetoIng** [mtex-toolbox/mtex#1007] [Ask Anything] [answered]
Hi,

I'm somewhat new to the whole field so please be patient  :-).

Right now I'm trying to calculate local maxima from ODF maps. The ODF map is estimated from EBSD data by using the function `calcDensity(ebsd(mineral).orientations)`. 

My problem, which is basically a lack of understanding, is how to get the Miller plane dependent ODF Map and maxima locations? 

Regards,
Christian

**Comment by lfgmorales** [comment, marked as answer]
Hi Christian

I am not sure I understand you correctly, but do you want to plot pole figures showing the orientations from your ebsd map? Or do you wan to have ODF sections?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 5 May 2021, at 09:54, cbetoIng ***@***.***> wrote:

﻿

Hi,

I'm somewhat new to the whole field so please be patient :-).

Right now I'm trying to calculate local maxima from ODF maps. The ODF map is estimated from EBSD data by using the function calcDensity(ebsd(mineral).orientations).

My problem, which is basically a lack of understanding, is how to get the Miller plane dependent ODF Map and maxima locations?

Regards,
Christian

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1007>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZIQGULCS3CMW7TIZDTMD2RXANCNFSM44EFDLIQ>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1007", "url": "https://github.com/mtex-toolbox/mtex/discussions/1007", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by cbetoIng** [reply]
Hi Luiz,

plotting the pole figures from EBSD-data by estimating the ODF is working as intended. I'm also able to calculate the local maxima of the ODF and plot them using  `plotPDF(odf, Miller({0,0,1},ebsd(mineral).CS), 'antipodal', 'contourf')`.

The local maxima of the ODF are used to calculate rotations about the Z, Y and X axis. Sometimes a maximum in (001) is chosen for rotation angle calculation and sometimes one in (110). 
My problem is, that by using 
`odf = calcDensity(ebsd(mineral).orientations,'silent'); `
`[m,ori] = max(odf,'numLocal',3); ` 
I get maxima that are different from what I can read from the plots. I assume that I have to perform some kind of projection of the maxima onto my Miller planes to calculate the transformed maxima and hence the correct rotation angles? But how?

Regards,
Christian

![image](https://user-images.githubusercontent.com/80693462/117251590-26b05980-ae45-11eb-83cf-d826e1806101.png)

**Reply by kilir** [reply]
Hi,
also, the maxima obtained from the ODF (actually an orientation where the highest density is located) need not to coincide with from pole figures, since the former obey crystal symmetry relations, the latter need not.
Hope that helps.
Cheers,
Rüdiger

**Reply by cbetoIng** [reply]
Hi Rüdiger,

thanks for the hint. Is there any way to calculate the maxima location with respect to the Miller indices? Somehow the `plotODF()` function is able to position the maxima at the corrent place when plotting. So it should be possible I assume.

Regards,
Christian

**Reply by kilir** [reply]
Hi Christian,
I'm not entirely sure what you mean by that, but if you want to get the maxima of individual polefigures, you can use `calcPDF` which will give you a spherical function and then find the maxima with `S2Fun/max()`.
Cheers,
Rüdiger

**Reply by cbetoIng** [reply]
Hi Rüdiger,

this is, what I was looking for. Thank you very much...highly appreciated.

Regards,
Christian
